linode/manager
# feat: [M3-5655] - Allow user to download DNS zone file

## Description 📝
This enables user to Download DNS Zone file. 

| Before  | After   |
| ------- | ------- |
| ![image](https://user-images.githubusercontent.com/119517080/235699906-961cf306-3955-4206-ab6c-631fc6ca48e5.png) | ![image](https://user-images.githubusercontent.com/119517080/235700184-fea765a7-cb7a-4421-b3c2-ca24d8093b4c.png) |

## How to test 🧪

- Navigate to http://localhost:3000/domains
- Click on any domain.
- Able to see "Download DNS Zone file" link type button.
- Upon clicking, should be able to download the DNS zone file.
